### PR TITLE
Feature/tools version update

### PR DESCRIPTION
**Pull request by lkindrat-xmos**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,18 +16,20 @@ RUN apt-get update && apt-get install -y \
...
 #   requires connection to XMOS network at build and run time
 #   if not possible, find another way to install the tools
-RUN mkdir -m 777 /XMOS && cd /XMOS \
-    && wget -q https://github0.xmos.com/raw/xmos-int/get_tools/master/get_tools.py \
-    && chmod a+x get_tools.py \
+RUN mkdir -m 777 /XMOS
+ADD https://github0.xmos.com/raw/xmos-int/get_tools/master/get_tools.py /XMOS/
+RUN cd /XMOS \
+    && chmod 755 get_tools.py \
     && echo "export MODULES_SILENT_SHELL_DEBUG=1\nexport MODULEPATH=/XMOS/modulefiles:/XMOS/template_modulefiles\nexport PATH=$PATH:/XMOS" \
```
Comment (keithm-xmos) [inline Dockerfile]: The `MODULES_SILENT_SHELL_DEBUG=1\nexport MODULEPATH=/XMOS/modulefiles:/XMOS/template_modulefiles\nexport` bits should no longer be needed.  

`"export PATH=$PATH:/XMOS"` should be all that is needed.

File: Jenkinsfile [hunk 2]
```diff
@@ -64,10 +64,12 @@ pipeline {
         }
         stage("Build") {
             steps {
-                // below is how we can activate the tools
-                sh """pushd /XMOS/tools/${params.TOOLS_VERSION}/XMOS/xTIMEcomposer/${params.TOOLS_VERSION} && . SetEnv && popd &&
+                // below is how we can activate the tools, NOTE: xTIMEcomposer -> XTC at tools 15.0.5
+                // sh """pushd /XMOS/tools/${params.TOOLS_VERSION}/XMOS/xTIMEcomposer/${params.TOOLS_VERSION} && // 
+                sh """pushd /XMOS/tools/${params.TOOLS_VERSION}/XMOS/XTC/${params.TOOLS_VERSION} &&
```
Comment (keithm-xmos) [inline Jenkinsfile]: The pushd + popd around SetEnv is no longer needed.  You now only need to:

`. /XMOS/tools/${params.TOOLS_VERSION}/XMOS/XTC/${params.TOOLS_VERSION}/SetEnv && . activate etc...`

**Review by keithm-xmos — COMMENTED** [review]
Made a few suggestions based on Brennan's recommendations.

**Review by BrennanGit — APPROVED** [review]
I'd definitely agree with all of Keiths comments. Another thing you could do to skip the whole docker caching issue is to move get_tools entirely out of it: ignore the centralised location mode of get_tools and just have a line like this instead to install it relative to the workspace:

```
sh "curl https://github0.xmos.com/raw/xmos-int/get_tools/master/get_tools.py | python - 15.0.5
```

It does make the docker image less plug and play but also makes it clear how you might update the tools if you view the jenkinsfile as a kind of executable example.

**Review by keithm-xmos — APPROVED** [review]

**Pull request merged**